leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/LeshanServerBuilder.java

```diff
@@ -62,6 +68,17 @@
 
     private NetworkConfig networkConfig;
 
+    /**
+     * <p>
+     * Set the address/port for unsecured CoAP Server.
+     * </p>
+     * 
+     * By default a wildcard address and the default CoAP port(5683) is used
+     * 
+     * @param hostname The address to bind. If null wildcard address is used.
+     * @param port A valid port value is between 0 and 65535. A port number of zero will let the system pick up an
+     *        ephemeral port in a bind operation.
```

Comment:
add a `@throws IllegalArgumentException` for port out of legal range?
add a `@return` tag?